app/controllers/nonsipp/membersurrenderedbenefits/SurrenderedBenefitsMemberListController.scala

```diff
@@ -254,7 +356,48 @@ object SurrenderedBenefitsMemberListController {
       buttonText = "site.saveAndContinue",
       details = None,
       onSubmit = controllers.nonsipp.membersurrenderedbenefits.routes.SurrenderedBenefitsMemberListController
-        .onSubmit(srn, page, mode)
+        .onSubmit(srn, page, mode),
+      optViewOnlyDetails = if (mode == ViewOnlyMode) {
```

Comment:
Hi @cyeniceri, here we can use 'mode.isViewOnlyMode' , instead of this mode == ViewOnlyMode